src/main/clojure/cemerick/pomegranate.clj

```diff
@@ -22,11 +22,17 @@
     (doto (.setAccessible true))
     (.invoke obj (into-array Object args))))
 
+(defn base-classloader []
+  (let [^DynamicClassLoader cl (RT/baseLoader)]
+    (if-let [^DynamicClassLoader parent (.getParent cl)]
```

Comment:
currently `clojure.main/repl` creates a new DCL which is sucessfully modified after 1 nrepl session, but then it's GC'ed and folowing `require` in same repl client fails

Reply:
Can you explain this in a bit more details?